pkg/gatewayserver/scheduling/scheduler.go

```diff
@@ -66,19 +67,32 @@ type RTTs interface {
 
 // NewScheduler instantiates a new Scheduler for the given frequency plan.
 // If no time source is specified, the system time is used.
-func NewScheduler(ctx context.Context, fp *frequencyplans.FrequencyPlan, enforceDutyCycle bool, timeSource TimeSource) (*Scheduler, error) {
+func NewScheduler(ctx context.Context, fp *frequencyplans.FrequencyPlan, enforceDutyCycle bool, scheduleAnytimeDelayMs uint32, timeSource TimeSource) (*Scheduler, error) {
+	logger := log.FromContext(ctx)
 	if timeSource == nil {
 		timeSource = SystemTimeSource
 	}
+
+	scheduleAnytimeDelay := time.Duration(scheduleAnytimeDelayMs) * time.Millisecond
+	if scheduleAnytimeDelay <= ScheduleTimeShort {
+		scheduleAnytimeDelay = ScheduleTimeLong
```

Comment:
- I think the sane minimum is `ScheduleTimeShort`
- If the value is zero/nil, default to `ScheduleTimeLong`

Reply:
Should be OK now